docs/grammar/SolidityLexer.g4

```diff
@@ -314,7 +315,7 @@ YulDecimalNumber: '0' | ([1-9] [0-9]*);
 YulStringLiteral:
 	'"' DoubleQuotedStringCharacter* '"'
 	| '\'' SingleQuotedStringCharacter* '\'';
-
+YulHexStringLiteral: HexString;
```

Comment:
I'm by the way not entirely clear on when and why antlr sometimes allows doing stuff like this "cross-mode" and when and why it doesn't - but since it seems to work ;-)...

Reply:
In https://677023-40892817-gh.circle-artifacts.com/0/docs-html/grammar.html#a4.SolidityLexer.YulHexNumber it doesn't link properly like this... not a big deal, though, but maybe we should look into that at *some* point...